src/framework/lib/genesisframework/utils.rb

```diff
@@ -44,9 +44,12 @@ def self.facter
         end
       end
 
-      def self.log subsystem, message
-        logline = subsystem.to_s + " :: " + message
-        puts logline
+      def self.log subsystem, message, level = nil
+        logline  = subsystem.to_s + " :: " + message
+
+        # Format the output on basic STDOUT
+        severity = level.nil? ? 'INFO' : level
```

Comment:
`INFO` would be the default for the raw output on std.
I didn't wanted to overwrite the logging module's  default.